### PR TITLE
Document Docker Desktop alternatives for Linux

### PR DESCRIPTION
**Pull request by BjoernAtBosch**
## Issue ticket number and link

AB#13530

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

**Review by mo84hi — APPROVED** [review]

File: content/en/docs/tutorials/quickstart/container_runtime.md [hunk 1]
```diff
@@ -8,11 +8,15 @@ description: >
 
 In the past the recommended runtime would for sure be [Docker Desktop](https://www.docker.com/products/docker-desktop). But since Docker Inc. changed their license model it is fair enough for an open source project to look for free alternatives.
 
-# macOS
+## Linux
+
+The obvious (and our recommended) alternative to Docker Desktop on Linux is [Docker Engine](https://docs.docker.com/engine/), a pure CLI-based solution available for most popular Linux distributions licensed under the Apache License, version 2.0. Installation instructions can be found [here](https://docs.docker.com/engine/install/).
```
Comment (MP91) [inline content/en/docs/tutorials/quickstart/container_runtime.md:13]: isn't the docker engine also part of docker desktop? So basically Docker Engine is not really an alternative but you can use it without Docker Desktop or? If so please adapt this section accordingly.

File: content/en/docs/tutorials/quickstart/container_runtime.md [hunk 2]
```diff
@@ -39,6 +43,18 @@ Docker Desktop uses 5 cores and 12 GB of RAM by default on an M1 MacBook Pro. Th
 
 That's all you have to do. After these few steps you can go on with the devcontainer setup.
 
-### Drawbacks
+#### Drawbacks
+
+The only drawback noticed so far is, that K9S is not working properly on M1 Macs. Since the container runtime and deployment are working also without K9S, this is just a minor issue. Nevertheless, the team is working on a solution.
+
+## Microsoft Windows
```
Comment (MP91) [inline content/en/docs/tutorials/quickstart/container_runtime.md:50]: Should we also mention the possibility to setup a virtual machine and use the linux way?

File: content/en/docs/tutorials/quickstart/container_runtime.md [hunk 3]
```diff
@@ -39,6 +43,18 @@ Docker Desktop uses 5 cores and 12 GB of RAM by default on an M1 MacBook Pro. Th
...
+
+There is currently no recommended alternative for Windows except using GitHub codespaces, a cloud-based development environment.
+
+## Other alternatives
+
+Besides our recommendations above there are further alternatives, that were not evaluated by us so far or we decided, not to give our recommendation so far.
```
Comment (MP91) [inline content/en/docs/tutorials/quickstart/container_runtime.md:56]: What about something like this?
```suggestion
Besides our recommendations above, there are further alternatives, which are not yet evaluated or have some other drawbacks, blocking a recommendation. 
```

**Review by MP91 — APPROVED** [review]
LGTM

**Pull request merged**